.evergreen/.evg.yml

```diff
@@ -194,7 +194,7 @@ functions:
   "upload test results":
     - command: attach.xunit_results
       params:
-        file: ./src/*/build/test-results/TEST-*.xml
+        file: ./src/*/build/test-results/test/TEST-*.xml
```

Comment:
The patch changed when we upgraded to Gradle 3.5 back in 2017.

Reply:
:mag: well spotted!